HISTORY.rst

```diff
@@ -79,7 +79,11 @@ Unreleased Changes
       middle clicking will close that tab as expected.
       (`#1261 <https://github.com/natcap/invest/issues/1261>`_)
     * Updated InVEST logo to use new version with registered trademark symbol.
-      (https://naturalcapitalproject.stanford.edu/invest-trademark-and-logo-use-policy)
+      (`InVEST TM and Logo Use Policy 
+       <https://naturalcapitalproject.stanford.edu/invest-trademark-and-logo-use-policy>`_)
```

Comment:
@davemfish this look right to me and I'm not sure why it's not rendering correctly when viewing in the rich diff.

Reply:
@dcdenu4 it looks like the new RST-linter (awesome!) is flagging these lines

Reply:
Yep, it was an indent problem! Fixed!